src/translate/pddl_parser/warning.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+
+PRINTED_WARNINGS = set()
+
+def print_warning(msg):
+    global PRINTED_WARNINGS
```

Comment:
We don't need global any more. The access here is not different from how we access other names from the global namespace, such as "sys" and "print". The old code needed "global" because it needed to rebind the global variable (use "="), but now we just call methods.

Reply:
Ah I didn't realize that assignment in particular is the problem. I changed it (and also changed the all-caps to no caps)